### PR TITLE
Milestone 5: Styling

### PR DESCRIPTION
**Pull request by AsumVictor**
# Milestone 5: Styling
In this PR, I have added styles and animation to the book app

 ## Type of change
 * [ ] use Tailwind CSS classes to style components

 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linter error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

File: src/index.css
Comment (joskalenda) [inline src/index.css:1]: - [x] I really appreciate your work, you have done an amazing job, dear @AsumVictor, however, before your project gets approved, please make sure you address these design modifications first, compare the 2 images, and bring necessary changes to your design to match the original template

YOUR DESIGN



FIGMA TEMPLATE



- [x] If believe my eyes, the font family you are using on your application is different from the correct one on the original template, take to change it, please `font-family: Montserrat;`

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @AsumVictor , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| IMAGE |

## Highlights

- [ ].. ✅ Linters
- [ ].. ✅ Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css
Comment (AsumVictor) [reply src/index.css:1]: THank you I have added the appropriate font and styles

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @AsumVictor  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, that has a failed linter check, Currently your Stylint is failing. You should fix this before requesting a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by V-Blaze — APPROVED** [review]
Hi @AsumVictor  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- UI looks great💯
-  Correct Git flow👍
-  Good communication with the reviewer🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsumVictor** [comment]
Thank you very much for your time to review the project.

**Pull request merged**